readthedocs/proxito/middleware.py

```diff
@@ -126,6 +126,13 @@ class ProxitoMiddleware(MiddlewareMixin):
 
     """The actual middleware we'll be using in prod."""
 
+    skip_views = (
```

Comment:
This could use a comment explaining why they can be skipped (basically what's in the PR)